### PR TITLE
fix dataset definition

### PR DESCRIPTION
**Pull request by sgratzl**

### CODE REVIEW

File: data/index.json [hunk 1]
```diff
@@ -177,11 +177,12 @@
     "name": "big-decent-clipped-38",
     "type": "table",
     "id": "big-decent-clipped-38",
+    "separator": ",",
     "description": "",
     "creator": "Alexander Lex",
     "path": "data_38.csv",
     "size": [
-      19,
+      102,
```
Comment (ngehlenborg) [inline data/index.json]: I think that the server should take care of determining the size of the data set. Setting this manually is error prone and tedious.

Comment (cnobre) [inline data/index.json]: Is there a way to do this Sam?

Comment (alexsb) [inline data/index.json]: I think the plan is to have the importer take care of this. If we're not using the importer though, we'll have to do it manually. 

The bizarre thing is that this file was created with the importer to my knowledge.

**Comment by cnobre** [comment]
It was. which is why I'm not sure why Sam said the data desc couldn't be created automatically.

File: data/index.json (hunk 1 above)
Comment (cnobre) [inline data/index.json]: It was. which is why I'm not sure why Sam said the data desc couldn't be created automatically.

**Pull request merged**